templates/singleton/boil_table_names.go.tpl

```diff
@@ -7,3 +7,24 @@ var TableNames = struct {
 	{{titleCase $table.Name}}: "{{$table.Name}}",
 	{{end -}}
 }
+
+var TypeNameToTableName = map[string]string {
```

Comment:
Are these useful for people? Or for the new mechanism?

Reply:
I think it's generally useful, but also specifically used when building the join query mod.